### ISSUE TITLE
tokenNotExpired() is not defined

### ISSUE THREAD

**Issue by kksrini89** [auth0/angular2-jwt#369]
After successful login, tokenNotExpired always returns false. So it doesn't redirect the secured page with angular guard.

But it showed me tokenNotExpired is not defined even though I have given reference using 
`import { tokenNotExpired } from 'angular2-jwt';`

Any Idea pls?

**NOTE**: I manually passed stored localstorage token name to tokenNotExpired method. It resolves the problem but I don't think this is the right way! 
```
public isAuthenticated() : boolean{
   return tokenNotExpired('id_token');
}
```

**Comment by chenkie** [comment]
I'm a bit confused by your mention of `tokenNotExpired` not being defined. Are you able to access the function in your app?

Under the hood, `tokenNotExpired` is looking for a token called `'token'`. You can set your default token name to be `id_token` in your configuration if you like.

```ts
export function authHttpServiceFactory(http: Http, options: RequestOptions) {
  return new AuthHttp(new AuthConfig({
    tokenName: 'id_token'
  }), http, options);
}
```

**Comment by kksrini89** [comment]
@chenkie  My problem solved by replacing the token. But anyhow I wondering why AUTH_PROVIDERS are not working from "angular-jwt" whereas we use exported function to resolve this problem.

**Comment by chenkie** [comment]
It has to do with functions being statically analyzable when the app is booted up. You can try using the `AuthModule` in your imports but there may be issues. Things will be fixed up for a v1.0 release.

**Comment by kksrini89** [comment]
Thank you for your quick help!
